src/DatatypeChannels.ASB/DatatypeChannels.ASB.fsproj

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
Not 6?

Reply:
I thought I could get a way with .NET 5 as the target for the lib while using F# 6 compiler to do that `task` majik... but I haven't tried using the results from .NET 5.
May have to come back to it later.